test/ui-test/testSuites/global_shared/steps/commonInitSteps.py

```diff
@@ -57,113 +57,137 @@ def context_init(context, testSettings, screenshot_on_fail = True):
 
     context.userData[_fixtures_root] = os.path.join(joined_path, "fixtures/")
 
+
 def a_first_time_user_lands_on(context):
-    filesMngr.erase_directory(context.userData[_status_data_folder])
-    common.start_application(context.userData[_aut_name])
+    common.start_application()
+
 
 def a_user_starts_the_application_with_a_specific_data_folder(context, data_folder_path):
```

Comment:
There is something wrong with the scenarios that uses this statement (particular status folder). 

I.e: Check `suite_messaging/tst_groupChat` to reproduce:

![Screenshot 2023-05-18 at 14 04 12](https://github.com/status-im/status-desktop/assets/97019400/fc2c19cf-5a75-4159-8fb1-42504d6372a3)

**NOTE**: This scenario can fail bc it is marked as `relyon-mailserver` but should fail in another verification step, like when trying to check the other user chat, not in this first step where the app is still pending to be launched.

Reply:
> There is something wrong with the scenarios that uses this statement (particular status folder).
> 
> I.e: Check `suite_messaging/tst_groupChat` to reproduce:
> 
> ![Screenshot 2023-05-18 at 14 04 12](https://user-images.githubusercontent.com/97019400/239228603-fc2c19cf-5a75-4159-8fb1-42504d6372a3.png)
> 
> **NOTE**: This scenario can fail bc it is marked as `relyon-mailserver` but should fail in another verification step, like when trying to check the other user chat, not in this first step where the app is still pending to be launched.

Tested and, at least, the folder interaction works fine. However it seems the `suite_messaging` is not up to date with new app changes, so I've created a new task for that: https://github.com/status-im/status-desktop/issues/10781